pylint/checkers/typecheck.py

```diff
@@ -1001,8 +1001,17 @@ def _compiled_generated_members(self) -> tuple[Pattern[str], ...]:
 
     @only_required_for_messages("keyword-arg-before-vararg")
     def visit_functiondef(self, node: nodes.FunctionDef) -> None:
-        # check for keyword arg before varargs
+        """Check for keyword arg before varargs.
```

Comment:
Bit of a nit but let's keep this as a comment. That makes it easier to later add another check to this `visit` method.

Reply:
Sure! Let me know if it still doesn't look right.